test_runtime.sh

```diff
@@ -79,8 +79,17 @@ cp runtimetest ${TESTDIR}
 
 oci-runtime-tool generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
 
-TESTCMD="${RUNTIME} start $(uuidgen)"
+conID=$(uuidgen)
+
+CREATECMD="${RUNTIME} create ${conID}"
+TESTCMD="${RUNTIME} start ${conID}"
 pushd $TESTDIR > /dev/null
+if ! ${CREATECMD}; then
+	info "Runtime is ${RUNTIME}, no need to create before start"
+else
+	info "Runtime is ${RUNTIME}, need to create before start"
+fi
```

Comment:
when exitcode of command "${RUNTIME} create ${conID}" is 1, runtime has no command "create" continue to run command "start". If exitcode of command "${RUNTIME} create ${conID}" is -1, runtime create container fail and print error. but I don't know how to get exitcode of command "${RUNTIME} create ${conID}".